onnx_utils/requirements.txt

```diff
@@ -1,7 +1,7 @@
 tqdm~=4.62.3
 tensorflow~=2.7.0
-torch~=1.10.0
-torchvision~=0.11.1
+torch==1.10.0
+torchvision==0.11.1
```

Comment:
why the freezing of the version? if it's not working with 1.10.x we should check it and see why, not freeze version and ignore it.